credentials/ssocreds/os.go

```diff
@@ -0,0 +1,9 @@
+// +build !windows
+
+package ssocreds
+
+import "os"
+
+func getHomeDirectory() string {
+	return os.Getenv("HOME")
+}
```

Comment:
Does this duplicate the behavior used by the shared config file loading for the default file name?  I think there are a few additional places the Windows home can be. Should these be considered for the sso creds file as well?

Reply:
Yes this logic is the same, in the `os_windows.go` file the implementation is to use the `USERPROFILE` environment variable. I opted for the build file strategy rather then the `runtime.GOOS` check that the shared_config currently usses.

Reply:
+1 build tag instead of runtime.